src/components/slider/slider.ts

```diff
@@ -47,6 +47,11 @@ export class MdSlider implements AfterContentInit {
   @HostBinding('attr.aria-disabled')
   disabled: boolean = false;
 
+  /** Whether or not to show the thumb label. */
+  @Input('thumb-label')
+  @BooleanFieldValue()
+  private _thumbLabel: boolean = false;
```

Comment:
This can't be `private` if you're referencing it in a template when doing offline compile. Just remove the `private` and leave the underscore.